modules/dns_common/dns_common.rb

```diff
@@ -170,10 +170,10 @@ def aaaa_record_conflicts(fqdn, ip)
       end
     end
 
-    def cname_record_conflicts(fqdn, target)
-      current = resolver.getresources(fqdn, Resolv::DNS::Resource::IN::CNAME)
+    def cname_record_conflicts(fqdn, host_alias)
+      current = resolver.getresources(host_alias, Resolv::DNS::Resource::IN::CNAME)
```

Comment:
It works already, but from my point of view this signature is incorrect.

`server.create_cname_record(fqdn, value)`
Actual CNAME implementations `fqdn` refers to host alias, while `value` refers to the true `fqdn`(`target`).

It makes more sense to use `fqdn=host.example.com, value=alias.example.com`, like IP attribution, `fqdn=host.example.com, value=10.0.0.1`.

Why CNAME is the only that would does the opposite?